server/services/caseloadService.ts

```diff
@@ -139,14 +143,17 @@ export default class CaseloadService {
             },
           }
         }
-        return {}
+        return { nomisRecord: offender }
```

Comment:
this is thrown away after the subsequent filter but passing it through so we store some additional contextual information along with exclusion